src/components/popup/details/vendors/vendors.test.js

```diff
@@ -1,88 +1,100 @@
 /* eslint-disable react/jsx-no-bind */
```

Comment:
@LassaLek This file should be updated or deleted given that the functionality to `select`, `accept`, `reject`, and `viewing` vendors was moved to another file (`details.jsx`, I believe)